### PR TITLE
VAGOV-2014: make q-a section h2s look like h3s

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
- for qa sections and titles, make the h2 looks like an h3
- added class for that style

## Testing done
- locally with staging data
- this page has both qa section and qa accordion headers: http://localhost:3001/drupal/health-care/eligibility/

## Screenshots
localhost_3001_drupal_health-care_eligibility_

## Acceptance criteria
- https://va-gov.atlassian.net/browse/VAGOV-2014

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/sass/static-pages.scss [hunk 1]
```diff
@@ -77,3 +77,10 @@
     display: none;
   }
 }
+
+
+// Makes anything look like an h3
+.h3 {
```
Comment (ahay-agile6) [inline src/applications/static-pages/sass/static-pages.scss]: i wasn't _exactly_ sure how to approach this - will there be more that need this style? do i need to add the font-family? it works for this immediate need

File: src/applications/static-pages/sass/static-pages.scss [hunk 2]
```diff
@@ -77,3 +77,10 @@
...
+  font-size: 2rem;
+  font-weight: 700;
+}
```
Comment (ncksllvn) [inline src/applications/static-pages/sass/static-pages.scss]: I think we actually have `.heading-level-3` for this 🙂 I think it's missing from our design system docs though.

@jaredcunha-usds Do you think it makes sense to include those headings classes under [utilities](https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/)?

File: src/applications/static-pages/sass/static-pages.scss (hunk 1 above)
Comment (jbalboni) [reply src/applications/static-pages/sass/static-pages.scss]: You might see if the `heading-level-3` class works for this.

**Review by jbalboni — APPROVED** [review]
One suggestion, but otherwise LGTM

File: src/applications/static-pages/sass/static-pages.scss (hunk 2 above)
Comment (jaredcunha-usds) [reply src/applications/static-pages/sass/static-pages.scss]: @ncksllvn - There is a `font-size` utility. I added some redundant classes to reflect that the font size refers to base styles of specific headings: https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/font-size

For these types of utilities, I tend to prefer the naming convention that describes exactly the CSS property it uses.

**Pull request merged**